scripts/cubic/bootstrap-fmriprep.sh

```diff
@@ -92,68 +144,54 @@ else
     datalad save -r -m "added input data"
 fi
 
-SUBJECTS=$(find inputs/data -type d -name 'sub-*' | cut -d '/' -f 3 )
+# dj: using subset for testing
+SUBJECTS=$(find inputs/data -type d -name 'sub-NDARZZ*' | cut -d '/' -f 3 )
 if [ -z "${SUBJECTS}" ]
 then
     echo "No subjects found in input data"
     # exit 1
 fi
 
-set +u
-CONTAINERDS=$2
-set -u
-#if [[ ! -z "${CONTAINERDS}" ]]; then
-cd ${PROJECTROOT}
-datalad clone ${CONTAINERDS} pennlinc-containers
-## Add the containers as a subdataset
-#datalad clone ria+ssh://sciget.pmacs.upenn.edu:/project/bbl_projects/containers#~pennlinc-containers pennlinc-containers
-# download the image so we don't ddos pmacs
-cd pennlinc-containers
-datalad get -r .
-# get rid of the references to pmacs
-set +e
-datalad siblings remove -s pmacs-ria-storage
-datalad siblings remove -s origin
-set -e
+#dj: should we change this part?
+CONTAINERDS=///repronim/containers
+datalad install -d . --source ${CONTAINERDS}
+datalad get containers/images/bids/bids-fmriprep--${VERSION}.sing
 
 cd ${PROJECTROOT}/analysis
-datalad install -d . --source ${PROJECTROOT}/pennlinc-containers
 
 ## the actual compute job specification
-cat > code/participant_job.sh << "EOT"
+cat > code/participant_job.sh << EOT
 #!/bin/bash
-#$ -S /bin/bash
-#$ -l h_vmem=25G
-#$ -l tmpfree=200G
-#$ -R y 
-#$ -l h_rt=24:00:00
-# Set up the correct conda environment
-source ${CONDA_PREFIX}/bin/activate base
-echo I\'m in $PWD using `which python`
 
+source $PRESCRIPT
```

Comment:
should this be sourced or inserted? and are the prescripts also tracked in datalad if not?

Reply:
ok, you're right it should be inserted, since I don't track it with datalad..